_config.php

```diff
@@ -60,7 +60,10 @@
     ->enablePlugins(array(
         'contextmenu' => null,
         'image' => null,
-        'sslink' => FRAMEWORK_ADMIN_DIR . '/client/dist/js/TinyMCE_sslink.js'
-    ));
+        'sslink' => FRAMEWORK_ADMIN_DIR . '/client/dist/js/TinyMCE_sslink.js',
```

Comment:
Can we make FRAMEWORK_ADMIN_DIR more risilient to being root, either by:

  - `ltrim($link, '/')` (probably just do this)
  - Not using FRAMEWORK_ADMIN_DIR at all (see https://github.com/silverstripe/silverstripe-asset-admin/commit/7bc13947054c08a892c410c3ba835011eeda8483#diff-8a7315557cd9f672e36f7e8f0ce2c29e for proof of concept).